PyApps/src/Plugins/result_plotter.py

```diff
@@ -98,20 +111,14 @@
 except:
   pass
 
-from QwtPlotImage_qt4 import *
-from QwtColorBar_qt4 import *
+from QwtPlotImage_qt5 import *
+from QwtColorBar_qt5 import *
```

Comment:
Avoid global namespace import. Qualify all the imports in this section

Reply:
done